style.css

```diff
@@ -0,0 +1,94 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+
+body {
+  align-items: center;
+  margin: 0;
+}
+
+.Top {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin-right: 25px;
+}
+
+.Top a {
+  margin-left: 25px;
+  padding-left: 5px;
+  padding-top: 10px;
+}
+
+nav {
+  margin-right: 30px;
+  padding-right: 5px;
+  padding-top: 10px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  align-items: center;
+  justify-content: space-between;
+  background-image: url("images/header-shapes\ mobile.png");
```

Comment:
- [ ] [OPTIONAL] The background in this section doesn't change with the screen width, please consider using ```background-size: cover;``` in the headline

![Screenshot_20220601_202032](https://user-images.githubusercontent.com/25227022/171485039-4dd92ce4-db4d-4028-9611-8aac538c51e4.png)